submission/races.go

```diff
@@ -71,8 +72,14 @@ func newSafeSubmissionState(groups ctpolicy.LogPolicyData) *safeSubmissionState
 	}
 	if baseGroup, ok := groups[ctpolicy.BaseName]; ok {
 		s.minGroups = baseGroup.MinOperators
+		switch {
+		case s.minGroups == 2:
+			s.maxSubmissionsPerGroup = 1
+		case s.minGroups == 3:
+			s.maxSubmissionsPerGroup = 2
+		}
```

Comment:
The maximum number of SCTs per log operator comes from the CT policy requirements. Having these logics in the policy package would be more appropriate.

Reply:
Makes sense. I have moved the decision for setting the maxSubmissionsPerGroup into the policy package.